xseed_qbit_cat_filter.sh

```diff
@@ -56,13 +56,12 @@ elif [[ -z "$TORRENT_CAT" ]]; then
   log_err "Category not specified for $TORRENT_PATH"
 fi
 
-log "[\033[1m$TORRENT_NAME\033[0m] [$TORRENT_CAT]"
 
 
 if [[ "$TORRENT_CAT" =~ ^($FILTERED_CAT)$ ]]; then
   xseed_resp=$(cross_seed_request "infoHash=$TORRENT_INFOHASH");
-  [ "$xseed_resp" != "204" ] && sleep 30 && xseed_resp=$(cross_seed_request "path=$TORRENT_PATH")
+  [ "$xseed_resp" != "204" ] && sleep 30 && xseed_resp=$(cross_seed_request "path=$TORRENT_PATH") && log "[\033[1m$TORRENT_NAME\033[0m] [$TORRENT_CAT]"
 elif [[ "$TORRENT_TRACKER" =~ ($FILTERED_TRACKER) ]]; then
   xseed_resp=$(cross_seed_request "infoHash=$TORRENT_INFOHASH");
-  [ "$xseed_resp" != "204" ] && sleep 30 && xseed_resp=$(cross_seed_request "path=$TORRENT_PATH")
+  [ "$xseed_resp" != "204" ] && sleep 30 && xseed_resp=$(cross_seed_request "path=$TORRENT_PATH") && log "[\033[1m$TORRENT_NAME\033[0m] [$TORRENT_TRACKER]"
```

Comment:
will this only log when the response from infohash search is 204 (not found) - I think this needs to go on the next (it's own) line?

Reply:
No it’ll log when the info hash search is not 204

Reply:
Sorry, I'm thinking announce endpoint.

send info hash search
if it receives 204 - which is success - then it does not log at all.

It should instead log outside of the condition that infohash fails